lib/util.js

```diff
@@ -417,6 +456,59 @@ function formatArray(ctx, value, recurseTimes, visibleKeys, keys) {
 }
 
 
+function formatSet(ctx, value, recurseTimes, visibleKeys, keys) {
+  var output = [];
+  value.forEach(function(v) {
+    var str = formatValue(ctx, v,
+                          recurseTimes === null ? null : recurseTimes - 1);
+    output.push(str);
+  });
```

Comment:
Nit: we should be able to do something like `for (var v of value) {` here instead of creating a closure.